store/src/java/com/zimbra/cs/service/account/ChangePassword.java

```diff
@@ -109,8 +109,10 @@ public Element handle(Element request, Map<String, Object> context) throws Servi
         AuthToken at = AuthProvider.getAuthToken(acct);
```

Comment:
Can we move this in if block this will avoid generating one auth token.

AuthToken at = AuthProvider.getAuthToken(acct);